src/client/src/commons/form/profile/components/profileLayers/profileLayers.js

```diff
@@ -104,10 +104,13 @@ const ProfileLayers = props => {
       alertContext.error(t("first_add_layer_to_depth"));
     } else {
       setSelectedLithologicalDescription(null);
+      const newFromDepth = lithoDescQuery?.data?.at(-1)?.toDepth ?? 0;
       layers?.data?.length !== 0
         ? addMutation.mutate({
             stratigraphyId: selectedStratigraphyID,
-            fromDepth: lithoDescQuery?.data?.at(-1)?.toDepth ?? 0,
+            fromDepth: newFromDepth,
+            toDepth: layers.data.find(l => l.depth_from === newFromDepth)
```

Comment:
Q: Könnte hier `layers` bzw. `data` null sein? Kann es sein, dass `find` nichts findet?
Wenn ja jeweis den `?.` operator verwenden.

Reply:
Ich verstehe hier grad nicht was github macht, aber habe genau das in 18b8b4585f509be57bf31cbf375449b7f0a2f720
gefixed 🤔 .